### PR TITLE
Let me pass options please!

### PR DESCRIPTION
**Pull request by rodrigoimenes**
Dont force parse :ghost:

### CODE REVIEW

**Comment by rodrigoimenes** [comment]
@dabroek is this project deprecated?

File: dist/index.js [hunk 1]
```diff
@@ -104,17 +104,21 @@ var redisStore = function redisStore() {
     },
     get: function get(key, options, cb) {
       return new Promise(function (resolve, reject) {
+        var defaultOpts = { parse: true };
         if (typeof options === 'function') {
           cb = options;
+          options = defaultOpts;
         }
 
+        if (!options) defaultOpts = options;
```
Comment (dabroek) [inline dist/index.js]: I think you might have wanted to have this assignment turned around. Please also add some brackets while you're add it.

**Comment by dabroek** [comment]
Hi @rodrigoimenes! Thanks for your PR. This project is certainly not deprecated, I just haven't had loads of time to look at it, so please be patient with me.

**Comment by dabroek** [comment]
Can you please also add a test for this new functionality?

**Comment by kessiler** [comment]
@dabroek would be possible to get this merge if I fix your comments and add tests?

**Pull request closed without merging**